modules/data.atmosphere/R/align_met.R

```diff
@@ -47,6 +47,7 @@
 ##' @param n.ens  - number of ensemble members to generate and save
 ##' @param pair.mems - logical stating whether ensemble members should be paired in 
 ##'                    the case where ensembles are being read in in both the training and source data
+##' @param mems.train - (optional) ensemble identifiers so that the training data is read in a specific order                    
```

Comment:
This documentation is insufficient for a user to know what they're supposed to put in here or what exactly this optional flag does. Could you provide a bit more explanation in the Details? And maybe an Example?